packages/components/inline-message/src/inline-message.ts

```diff
@@ -77,16 +70,48 @@ export class InlineMessage extends ScopedElementsMixin(LitElement) {
       case 'warning':
         return 'octagon-exclamation-solid';
       case 'danger':
-        return 'triangle-exclamation-solid';
+        return 'diamond-exclamation-solid';
       default:
         return 'info';
     }
   }
 
-  override connectedCallback(): void {
-    super.connectedCallback();
+  /**
+   * If set, will remove the ability to dismiss the inline message by removing
+   * the close button.
+   * @default false
+   */
+  @property({ type: Boolean, reflect: true }) indismissible?: boolean;
+
+  /** @internal If the action is missing, we need to hide the action part. */
+  @property({ type: Boolean, attribute: 'no-action', reflect: true }) noAction = true;
+
+  /** @internal If the title is missing, the content needs to be placed where the title should be. */
+  @property({ type: Boolean, attribute: 'no-title', reflect: true }) noTitle = true;
+
+  get size(): InlineMessageSize {
+    return this.#size;
+  }
+
+  /**
+   * The size of the inline message.
+   * @default 'auto'
+   */
+  @property({ reflect: true })
+  set size(size: InlineMessageSize) {
```

Comment:
It looks like there is sth wrong with the `sm` and `md` sizes?

<img width="648" alt="image" src="https://github.com/user-attachments/assets/5af12859-e116-4619-acca-ae03e18ddc64" />

Also the close button should be aligned to top?

Reply:
Hmm, i don't think a title is meant to be used with the small size.

Reply:
I'll check with Danielle

Reply:
Yes, this behavior is as designed. We need to document this properly.